packages/cli/src/cmds/preview/depsChecker/cliAdapter.ts

```diff
@@ -8,6 +8,7 @@ import { IDepsAdapter, IDepsTelemetry } from "./checker";
 import CLIUIInstance from "../../../userInteraction";
 import cliLogger from "../../../commonlib/log";
 import { CliConfigOptions, UserSettings } from "../../../userSetttings";
+import { cliEnvCheckerTelemetry } from "./cliTelemetry";
```

Comment:
Why to import this module if it is not used in this file?

Reply:
fixed